From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Mon, 4 Apr 2016 20:43:47 -0400
Subject: fixup format warning

---
 asApp/src/save_restore.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/asApp/src/save_restore.c b/asApp/src/save_restore.c
index 4de4405..385f52e 100644
--- a/asApp/src/save_restore.c
+++ b/asApp/src/save_restore.c
@@ -1367,7 +1367,7 @@ STATIC int connect_list(struct chlist *plist, int verbose)
 	}
 	epicsSnprintf(SR_recentlyStr, STATUS_STR_LEN-1, "%s: %d of %d PV's connected", plist->save_file, n, m);
 	if (verbose) {
-		printf(SR_recentlyStr);
+		printf("%s", SR_recentlyStr);
 		printf("\n");
 	}
 
