From 6c5a4aeadeb08035a51ed22f7a3169b0f1564ac5 Mon Sep 17 00:00:00 2001
From: Daron Chabot <chabot@frib.msu.edu>
Date: Thu, 19 May 2016 15:32:16 -0400
Subject: [PATCH] install db and req files

---
 asApp/Db/Makefile | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/asApp/Db/Makefile b/asApp/Db/Makefile
index 983981d..5b2eb6f 100644
--- a/asApp/Db/Makefile
+++ b/asApp/Db/Makefile
@@ -10,7 +10,11 @@ include $(TOP)/configure/CONFIG
 #----------------------------------------------------
 # Create and install (or just install) into <top>/db
 # databases, templates, substitutions like this
-#DB += xxx.db
+DB += save_restoreStatus.db
+DB += configMenu.db
+DB += configMenu.req
+DB += configMenuNames.req
+DB += configMenu_settings.req
 
 #----------------------------------------------------
 # If <anyname>.db template is not named <anyname>*.template add
-- 
2.1.4

